vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -535,17 +600,9 @@ public VimeoAccount logIn(String email, String password) {
                 mVimeoService.logIn(getBasicAuthHeader(), email, password, Vimeo.PASSWORD_GRANT_TYPE,
                                     mConfiguration.scope);
 
-        VimeoAccount vimeoAccount = null;
-        try {
-            retrofit2.Response<VimeoAccount> response = call.execute();
-            if (response.isSuccessful()) {
-                vimeoAccount = response.body();
-            }
-        } catch (IOException e) {
-            ClientLogger.e("Exception during logIn: " + e.getMessage(), e);
-        }
+        VimeoAccount vimeoAccount = executeAccountCall(call);
 
-        saveAccount(vimeoAccount, email);
+        saveAuthenticatedUserAccount(vimeoAccount, email);
```

Comment:
Seems like we call this `email` here but something like username or account name above. Can we be consistent about how we refer to this piece of data?